lib/twirp/rails/configuration.rb

```diff
@@ -6,10 +6,11 @@ class Configuration
       # A lambda that accepts |rack_env, env| and is passed to Twirp::Service
       # See: https://github.com/twitchtv/twirp-ruby/wiki/Service-Hooks
       # for available hooks
-      attr_accessor :service_hooks
+      attr_accessor :service_hooks, :twirp_directories
```

Comment:
Worth making a separate `attr_accessor` call here so it can be documented separately.

Naming-wise, what about something like `load_paths` or `search_paths` or similar?

Reply:
Updated.